### PR TITLE
added missing "cassandra" db template to fix the "cassandra" support for jhipster kubernetes

### PR DESCRIPTION
**Pull request by govinda258**
added missing "cassandra" db template to fix the "cassandra" support for jhipster kubernetes

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/kubernetes/templates/db/cassandra.yml.ejs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        # These volume mounts are persistent. They are like inline claims,
+        # but not exactly because the names need to match exactly one of
+        # the stateful pod volumes.
+        volumeMounts:
+        - name: cassandra-data
+          mountPath: /cassandra_data
+  # These are converted to volume claims by the controller
+  # and mounted at the paths mentioned above.
+  # do not use these in production until ssd GCEPersistentDisk or other ssd pd
+  volumeClaimTemplates:
```
Comment (xetys) [inline generators/kubernetes/templates/db/cassandra.yml.ejs]: Please use the dynamic storage provisioning for storage. See the mongo StatefulSet as a reference

File: generators/kubernetes/templates/db/cassandra.yml.ejs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  - metadata:
+      name: cassandra-data
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      storageClassName: fast
+      resources:
+        requests:
+          storage: 1Gi
+---
+kind: StorageClass
```
Comment (xetys) [inline generators/kubernetes/templates/db/cassandra.yml.ejs]: Please remove the storage class. This is not the scope of the generator to define cluster storage classes.

**Comment by pascalgrimaud** [comment]
@govinda258 : thanks for your contribution. Can you have a look at the comments plz ?

**Comment by DanielFran** [comment]
@govinda258 Can you please review the comments?

**Comment by mraible** [comment]
Is anyone able to work on this PR to improve it and make it mergeable? If not, we should probably close it.

**Comment by pascalgrimaud** [comment]
Let's close this, as it's opened since 1 year and we have too few users with Cassandra

**Pull request closed without merging**